src/lib/core/redux/slices/localMedia.ts

```diff
@@ -298,6 +298,12 @@ export const reactSetDevice = createAppAsyncThunk(
                 { replaceStream: stream }
             );
 
+            const isAudioEnabled = selectIsMicrophoneEnabled(state);
+            stream.getAudioTracks().forEach((track) => (track.enabled = isAudioEnabled));
+
+            const isVideoEnabled = selectIsCameraEnabled(state);
+            stream.getVideoTracks().forEach((track) => (track.enabled = isVideoEnabled));
```

Comment:
Is it a possible scenario that someone has multiple video or audio tracks and not all of them should be muted/unmuted? If it is: this may be problematic as it seems to enable/disable all of them. 🙈

Reply:
This is copied from the pwa. I don't think it's ever been a problem there, so it should be fine